integration_tests/snap.snapcraft.yaml

```diff
@@ -0,0 +1,5 @@
+---
+description: A test part with snap/snapcraft.yaml
+maintainer: John Doe <john.doe@example.com>
+parts: [snap-testpart]
+origin: lp:~joetalbott/+junk/snap-testpart
```

Comment:
I agree that it's suboptimal.  I'm working on an update to move all the parser tests to use git repos generated on-the-fly on the local disk.  Should speed up integration tests a bit too.